docs/source/examples/index.rst

```diff
@@ -42,3 +42,11 @@ Modifying variables saved in the output file
    :maxdepth: 1
 
    custom_saving
+
+Defining custom YAML parameters
```

Comment:
can we collapse this header and the one above into "Modifying model input/output"